datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -0,0 +1,290 @@
+- name: openmetrics_endpoint
+  required: true
+  description: |
+    The URL exposing metrics in the OpenMetrics format.
+  value:
+    type: string
```

Comment:
does `telemetry` need to be documented: https://github.com/DataDog/integrations-core/blob/8f390e40f0f4309a9d4edb64964d5b76953e715f/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/scraper.py#L37

Reply:
good question. cc @CharlyF

Reply:
I think it should, the telemetry can be very helpful to debug if the endpoint is healthy.